Перевести события и коды безопасности

MorgoWizard opened the pull request:
Большая часть событий переведена, как и коды безопасности

MorgoWizard committed:
Перевести события и коды безопасности

Большая часть событий переведена, как и коды безопасности

semoro commented on code/modules/events/abductor.dm:
```diff
@@ -4,12 +4,12 @@
 	processing = 0 //so it won't fire again in next tick
 	if(!makeAbductorTeam())
-		message_admins("Abductor event failed to find players. Retrying in 30s.")
+		message_admins("Игроки на роли Абдукторов не были найдены. Повторная попытка через 30 секунд.")
```
Давайте не будем переводить сообщения админам

Zigamus replied:
А почему?

semoro commented on code/modules/events/alien_infestation.dm:
```diff
@@ -10,9 +10,9 @@
+		GLOB.event_announcement.Announce("Обнаружены неопознанные формы жизни на борту станции [station_name()]. Обезопасьте все наружные входы и выходы, включая трубопроводы и вентиляцию.", "ВНИМАНИЕ: НЕОПОЗНАННЫЕ ФОРМЫ ЖИЗНИ", new_sound = 'sound/AI/aliens.ogg')
 	else
-		log_and_message_admins("Warning: Could not spawn any mobs for event Alien Infestation")
+		log_and_message_admins("Внимание: не удалось создать существо для события 'Нашествие Чужих'")
```
Давайте не будем переводить сообщения админам

semoro commented on code/modules/events/apc_overload.dm:
```diff
@@ -19,9 +19,9 @@
+// уточнить про underfloor
 /datum/event/apc_overload/announce()
-	GLOB.event_announcement.Announce("Overload detected in [station_name()]'s powernet. Engineering, please check all underfloor APC terminals.", "Critical Power Failure", new_sound = 'sound/AI/attention.ogg')
+	GLOB.event_announcement.Announce("Зафиксирована перегрузка энергосети станции [station_name ()]. Инженерному отделу надлежит проверить все терминалы ЛКП под напольным покрытием.", "КРИТИЧЕСКИЙ СБОЙ СИСТЕМЫ ПИТАНИЯ", new_sound = 'sound/AI/attention.ogg')
```
лишний пробел между station_name и скобками `[station_name ()]`

semoro commented on code/modules/events/apc_overload.dm:
```diff
@@ -32,7 +32,7 @@
 
 	if(announce)
-		GLOB.event_announcement.Announce("Overload detected in [station_name()]'s powernet. Engineering, please check all underfloor APC terminals.", "Critical Power Failure", new_sound = 'sound/AI/attention.ogg')
+		GLOB.event_announcement.Announce("Зафиксирована перегрузка энергосети станции [station_name ()]. Инженерному отделу надлежит проверить все терминалы ЛКП под напольным покрытием.", "КРИТИЧЕСКИЙ СБОЙ СИСТЕМЫ ПИТАНИЯ", new_sound = 'sound/AI/attention.ogg')
```
лишний пробел между station_name и скобками `[station_name ()]`

semoro commented on code/modules/events/apc_short.dm:
```diff
@@ -20,8 +20,8 @@
-/datum/event/apc_short/announce()
-	GLOB.event_announcement.Announce("Overload detected in [station_name()]'s powernet. Engineering, please repair shorted APCs.", "Systems Power Failure", new_sound = 'sound/AI/attention.ogg')
+/datum/event/apc_short/announce() // закороченные - выведенные из строя
+	GLOB.event_announcement.Announce("Зафиксирована перегрузка энергосети станции [station_name ()]. Инженерному отделу надлежит проверить все замкнувшие ЛКП.", "СБОЙ СИСТЕМЫ ПИТАНИЯ", new_sound = 'sound/AI/attention.ogg')
```
лишний пробел между station_name и скобками `[station_name ()]`

semoro commented on code/modules/events/apc_short.dm:
```diff
@@ -32,7 +32,7 @@
 
 	if(announce)
-		GLOB.event_announcement.Announce("Overload detected in [station_name()]'s powernet. Engineering, please repair shorted APCs.", "Systems Power Failure", new_sound = 'sound/AI/attention.ogg')
+		GLOB.event_announcement.Announce("Зафиксирована перегрузка энергосети станции [station_name ()]. Инженерному отделу надлежит проверить все замкнувшие ЛКП.", "СБОЙ СИСТЕМЫ ПИТАНИЯ", new_sound = 'sound/AI/attention.ogg')
```
лишний пробел между station_name и скобками `[station_name ()]`

semoro commented on code/modules/events/brand_intelligence.dm:
```diff
@@ -5,16 +5,16 @@
 
 /datum/event/brand_intelligence/announce()
-	GLOB.event_announcement.Announce("Rampant brand intelligence has been detected aboard [station_name()], please stand-by. The origin is believed to be \a [originMachine.name].", "Machine Learning Alert")
+	GLOB.event_announcement.Announce("На борту станции [station_name ()] зафиксировано распространение цифрового торгового вируса, пожалуйста, будьте наготове. Вирус, предположительно, берет начало от \a [originMachine.name].", "ВНИМАНИЕ: ЦИФРОВОЙ ВИРУС")
```
лишний пробел между station_name и скобками `[station_name ()]`

semoro commented on code/modules/events/rogue_drones.dm:
```diff
@@ -18,12 +18,12 @@
 	var/msg
 	if(prob(33))
-		msg = "A combat drone wing operating out of the NSV Icarus has failed to return from a sweep of this sector, if any are sighted approach with caution."
+		msg = "Группа боевых дронов, оперируемых с борта ИКН ''Икар'', не вернулась с зачистки сектора. В случае контакта с дронами проявляйте осторожность."
```
Кавычки или одиночные `'` или `\"`

semoro commented on code/modules/events/rogue_drones.dm:
```diff
@@ -18,12 +18,12 @@
+		msg = "Группа боевых дронов, оперируемых с борта ИКН ''Икар'', не вернулась с зачистки сектора. В случае контакта с дронами проявляйте осторожность."
 	else if(prob(50))
-		msg = "Contact has been lost with a combat drone wing operating out of the NSV Icarus. If any are sighted in the area, approach with caution."
+		msg = "Потеряна связь с группой боевых дронов, оперируемых с борта ИКН ''Икар''. В случае контакта с дронами проявляйте осторожность."
```
Кавычки или одиночные `'` или `\"`

semoro commented on code/modules/events/rogue_drones.dm:
```diff
@@ -18,12 +18,12 @@
 	else
-		msg = "Unidentified hackers have targetted a combat drone wing deployed from the NSV Icarus. If any are sighted in the area, approach with caution."
-	GLOB.event_announcement.Announce(msg, "Rogue drone alert")
+		msg = "Неопознанные хакеры взломали систему контроля боевых дронов, оперируемых с борта ИКН ''Икар''. В случае контакта с дронами проявляйте осторожность."
```
Кавычки или одиночные `'` или `\"`

semoro commented on code/modules/events/rogue_drones.dm:
```diff
@@ -36,6 +36,6 @@
 
 	if(num_recovered > drones_list.len * 0.75)
-		GLOB.event_announcement.Announce("Icarus drone control reports the malfunctioning wing has been recovered safely.", "Rogue drone alert")
+		GLOB.event_announcement.Announce("Система контроля боевых дронов сообщает, что все единицы успешно вернулись на борт ''Икара''.", "ВНИМАНИЕ: СБОЙНЫЕ ДРОНЫ")
```
Кавычки или одиночные `'` или `\"`

semoro commented on code/modules/events/spider_terror.dm:
```diff
@@ -11,9 +11,9 @@
+		GLOB.command_announcement.Announce("Вспышка биологической угрозы 3-го уровня зафиксирована на борту станции [station_name()]. Всему персоналу надлежит сдержать ее распространение любой ценой!", "ВНИМАНИЕ: БИОЛОГИЧЕСКАЯ УГРОЗА", 'sound/effects/siren-spooky.ogg')
 	else
-		log_and_message_admins("Warning: Could not spawn any mobs for event Terror Spiders")
+		log_and_message_admins("Внимание: Не удалось создать существо для события ''Пауки Ужаса''")
```
Кавычки или одиночные `'` или `\"`

semoro commented on code/modules/events/traders.dm:
```diff
@@ -17,7 +17,7 @@ GLOBAL_LIST_INIT(unused_trade_stations, list("sol"))
 		return
 	if(seclevel2num(get_security_level()) >= SEC_LEVEL_RED)
-		GLOB.event_announcement.Announce("A trading shuttle from Jupiter Station has been denied docking permission due to the heightened security alert aboard [station_name()].", "Trader Shuttle Docking Request Refused")
+		GLOB.event_announcement.Announce("Торговому шаттлу со станции Юпитер было отказано в разрешении на стыковку из-за повышенной угрозы безопасности на борту [station_name()].", "Запрос на стыковку шаттла торговцев отклонен")
```
Со станции на орбите Юпитера наверное

semoro commented on code/modules/events/traders.dm:
```diff
@@ -55,14 +55,14 @@ GLOBAL_LIST_INIT(unused_trade_stations, list("sol"))
 				success_spawn = 1
 		if(success_spawn)
-			GLOB.event_announcement.Announce("A trading shuttle from Jupiter Station has been granted docking permission at [station_name()] arrivals port 4.", "Trader Shuttle Docking Request Accepted")
+			GLOB.event_announcement.Announce("Торговый шаттл со станции Юпитер получил разрешение на стыковку в порту прибытия 4 [station_name()].", "Запрос на стыковку шаттла торговцев принят")
```
Со станции на орбите Юпитера наверное

semoro commented on code/modules/events/vent_clog.dm:
```diff
@@ -6,7 +6,7 @@
 
 /datum/event/vent_clog/announce()
-	GLOB.event_announcement.Announce("The scrubbers network is experiencing a backpressure surge.  Some ejection of contents may occur.", "Atmospherics alert")
+	GLOB.event_announcement.Announce("Зафиксирован скачок обратного давления в системе вытяжных труб. Возможен выплеск содержимого.", "ВНИМАНИЕ: ГАЗЫ")
```
Выброс содержимого

semoro commented on code/modules/security_levels/security_levels.dm:
```diff
@@ -105,7 +105,7 @@ GLOBAL_DATUM_INIT(security_announcement_down, /datum/announcement/priority/secur
 
 			if(SEC_LEVEL_EPSILON)
-				GLOB.security_announcement_up.Announce("Central Command has ordered the Epsilon security level on the station. Consider all contracts terminated.","Attention! Epsilon security level activated!", new_sound = sound('sound/effects/purge_siren.ogg'))
+				GLOB.security_announcement_up.Announce("Центральным командованием был установлен код ЭПСИЛОН. Считайте, что все контракты расторгнуты.","ВНИМАНИЕ! КОД ЭПСИЛОН", new_sound = sound('sound/effects/purge_siren.ogg'))
```
`Центральным командованием был установлен код ЭПСИЛОН. Все контракты расторгнуты.`

semoro commented on code/modules/security_levels/security_levels.dm:
```diff
@@ -117,7 +117,7 @@ GLOBAL_DATUM_INIT(security_announcement_down, /datum/announcement/priority/secur
 
 			if(SEC_LEVEL_DELTA)
-				GLOB.security_announcement_up.Announce("The station's self-destruct mechanism has been engaged. All crew are instructed to obey all instructions given by heads of staff. Any violations of these orders can be punished by death. This is not a drill.","Attention! Delta security level reached!", new_sound = sound('sound/effects/deltaalarm.ogg'))
+				GLOB.security_announcement_up.Announce(" Механизм самоуничтожения станции задействован. Все члены экипажа обязан подчиняться всем указаниям, данными Главами отделов. Любые нарушения этих приказов наказуемы уничтожением на месте. Это не учебная тревога.","ВНИМАНИЕ! КОД ДЕЛЬТА!", new_sound = sound('sound/effects/deltaalarm.ogg'))
```
Пробел лишний в начале

semoro commented on code/modules/events/blob.dm:
```diff
@@ -5,9 +5,9 @@
+		GLOB.event_announcement.Announce("Вспышка биологической угрозы 5-го уровня зафиксирована на борту станции [station_name()]. Всему персоналу надлежит сдержать ее распространение любой ценой!", "ВНИМАНИЕ: БИОЛОГИЧЕСКАЯ УГРОЗА", 'sound/AI/outbreak5.ogg')
 	else
-		log_and_message_admins("Warning: Could not spawn any mobs for event Blob")
+		log_and_message_admins("ВНИМАНИЕ: Не удалось создать существо для проведения события 'Блоб'")
```
Админские сообщения переводить не надо

semoro commented on code/modules/events/meaty_ops.dm:
```diff
@@ -1,6 +1,6 @@
 /datum/event/meteor_wave/goreop/announce()
-	var/meteor_declaration = "MeteorOps have declared their intent to utterly destroy [station_name()] with their own bodies, and dares the crew to try and stop them."
-	GLOB.event_announcement.Announce(meteor_declaration, "Declaration of 'War'", 'sound/effects/siren.ogg')
+	var/meteor_declaration = "Метеориты заявили о своем намерении полностью уничтожить [station_name()] своими собственными телами. Осмелится ли экипаж остановить их?"
```
МБ `МетеоритыКо`

MorgoWizard committed:
Исправить перевод

semoro commented on code/modules/events/infestation.dm:
```diff
@@ -29,34 +29,34 @@
 		if(LOC_VAULT)
 			spawn_area_type = /area/security/nuke_storage
-			locstring = "the vault"
+			locstring = "убежище"
```
Хранилище

semoro commented on code/modules/events/traders.dm:
```diff
@@ -17,7 +17,7 @@ GLOBAL_LIST_INIT(unused_trade_stations, list("sol"))
 		return
 	if(seclevel2num(get_security_level()) >= SEC_LEVEL_RED)
-		GLOB.event_announcement.Announce("A trading shuttle from Jupiter Station has been denied docking permission due to the heightened security alert aboard [station_name()].", "Trader Shuttle Docking Request Refused")
+		GLOB.event_announcement.Announce("Торговому шаттлу с орбиты Юпитера было отказано в разрешении на стыковку из-за повышенной угрозы безопасности на борту [station_name()].", "Запрос на стыковку шаттла торговцев отклонен")
```
со станции Юпитер-6

semoro commented on code/modules/events/traders.dm:
```diff
@@ -55,14 +55,14 @@ GLOBAL_LIST_INIT(unused_trade_stations, list("sol"))
 				success_spawn = 1
 		if(success_spawn)
-			GLOB.event_announcement.Announce("A trading shuttle from Jupiter Station has been granted docking permission at [station_name()] arrivals port 4.", "Trader Shuttle Docking Request Accepted")
+			GLOB.event_announcement.Announce("Торговый шаттл с орбиты Юпитера получил разрешение на стыковку в порту прибытия 4 [station_name()].", "Запрос на стыковку шаттла торговцев принят")
```
со станции Юпитер-6

PapaLeroy commented on code/modules/events/alien_infestation.dm:
```diff
@@ -10,7 +10,7 @@
-		GLOB.event_announcement.Announce("Unidentified lifesigns detected coming aboard [station_name()]. Secure any exterior access, including ducting and ventilation.", "Lifesign Alert", new_sound = 'sound/AI/aliens.ogg')
+		GLOB.event_announcement.Announce("Обнаружены неопознанные формы жизни на борту станции [station_name()]. Обезопасьте все наружные входы и выходы, включая трубопроводы и вентиляцию.", "ВНИМАНИЕ: НЕОПОЗНАННЫЕ ФОРМЫ ЖИЗНИ", new_sound = 'sound/AI/aliens.ogg')
 	else
 		log_and_message_admins("Warning: Could not spawn any mobs for event Alien Infestation")
```
Внимание: Невозможно создать существо для событья Нашествие Чужих.

PapaLeroy commented on code/modules/events/alien_infestation.dm:
```diff
@@ -21,7 +21,7 @@
 
 	spawn()
-		var/list/candidates = SSghost_spawns.poll_candidates("Do you want to play as an alien?", ROLE_ALIEN, TRUE, source = /mob/living/carbon/alien/larva)
+		var/list/candidates = SSghost_spawns.poll_candidates("Вы хотите сыграть за Чужого?", ROLE_ALIEN, TRUE, source = /mob/living/carbon/alien/larva)
```
Вы хотите занять роль Чужого?
_Типа, везде так же прописано, чоб не тут тоже?_

PapaLeroy commented on code/modules/events/carp_migration.dm:
```diff
@@ -13,10 +13,10 @@
 	else
-		announcement = "Unknown biological entities have been detected near [station_name()], please stand-by."
-	GLOB.event_announcement.Announce(announcement, "Lifesign Alert")
+		announcement = "Неизвестные биологические объекты были обнаружены вблизи станции [station_name()], будьте наготове."
```
Вместо "обнаружено" стилистически корректно все-таки будет "зафиксировано"

PapaLeroy commented on code/modules/events/money_hacker.dm:
```diff
@@ -18,9 +18,9 @@ GLOBAL_VAR_INIT(account_hack_attempted, 0)
-	without intervention this attack will succeed in approximately 10 minutes. Required intervention: temporary suspension of affected account until the attack has ceased. \
-	Notifications will be sent as updates occur.<br>"
+	var/message = "Обнаружен взлом  (выполняется с момента [station_time_timestamp()]). Целью атаки является: Финансовый счет #[affected_account.account_number], \
+без вмешательства эта атака будет успешной примерно через 10 минут. Требуемое вмешательство: временно приостановтье любую активность затронутой учетной записи до тех пор, пока атака не прекратится. \
```
Требуется вмешательство - временно заморозьте учетную запись...

PapaLeroy commented on code/modules/events/spider_terror.dm:
```diff
@@ -47,7 +47,7 @@
-	var/list/candidates = SSghost_spawns.poll_candidates("Do you want to play as a terror spider?", null, TRUE, source = spider_type)
+	var/list/candidates = SSghost_spawns.poll_candidates("Вы хотите занять роль Паука Ужаса?", null, TRUE, source = spider_type)
 	if(length(candidates) < spawncount)
 		message_admins("Warning: not enough players volunteered to be terrors. Could only spawn [length(candidates)] out of [spawncount]!")
```
Внимание: недостаточно игроков заняли роль пауков Ужаса - всего [length(candidates)] из [spawncount]!

_Я хз, просто так это вставлю, потому что не знаю, воспринимают ли админские логи кириллицу._

MorgoWizard replied:
решили не переводить сообщения для администрации

MorgoWizard committed:
Сделать стилистические правки перевода

Исправил некоторые ошибки перевода. Произвёл некоторые стилистические изменения.
